adopters.md

```diff
@@ -7,31 +7,71 @@ category: adopters
-- [Ping++ (Mobile Payment)](https://www.crunchbase.com/organization/ping-5)
-- [Qunar.com (Travel)](https://www.crunchbase.com/organization/qunar-com)
-- [LY.com (Travel)](https://www.crunchbase.com/organization/ly-com)
-- [LinkDoc Technology (HealthTech)](https://www.crunchbase.com/organization/linkdoc-technology)
-- [WEIRUIDA (Clean Energy and Data Analytics)](https://pitchbook.com/profiles/company/161294-41)
-- [Yuanfudao (EdTech)](https://www.crunchbase.com/organization/yuanfudao)
-- [360 Financial (FinTech)](https://jinrong.360jie.com.cn/)
-- [China Telecom BestPay (FinTech)](https://www.bestpay.com.cn/global/oig/index.html)
-- [GAEA (Gaming)](http://gaea.com/en)
-- [YOOZOO GAMES (Gaming)](http://www.yoozoo.com/en)
-- [Seasun Games (Gaming)](http://www.seasungames.com/)
-- [FUNYOURS JAPAN (Gaming)](http://company.funyours.co.jp/)
-- [Hainan eKing Technology (Enterprise Technology)](https://www.crunchbase.com/organization/hainan-eking-technology)
-- [2Dfire (FoodTech)](http://www.2dfire.com/)
-- [Keruyun (FoodTech)](http://www.keruyun.com/en)
-- [G7 (Internet of Things)](https://www.english.g7.com.cn/)
-- [Yimian Data (Big Data)](https://www.yimian.com.cn)
-- [Qingdao Telaidian (Electric Car Charger)](https://www.teld.cn/)
-- [Wanda Internet Technology Group (Big Data)](http://www.wanda-tech.cn/en/)
+|Company| Industry| Success Story| 
+| --- | --- | --- |
+|[Mobike](https://en.wikipedia.org/wiki/Mobike)|Ridesharing|[English](https://www.pingcap.com/blog/Use-Case-TiDB-in-Mobike/); [Chinese](https://www.pingcap.com/cases-cn/user-case-mobike/)|
+|[Jinri Toutiao](https://en.wikipedia.org/wiki/Toutiao)|Mobile News Platform|[Chinese](https://www.pingcap.com/cases-cn/user-case-toutiao/)|
+|[Yiguo.com](https://www.crunchbase.com/organization/shanghai-yiguo-electron-business)|E-commerce|[English](https://www.datanami.com/2018/02/22/hybrid-database-capturing-perishable-insights-yiguo/); [Chinese](https://www.pingcap.com/cases-cn/user-case-yiguo)|
+|[Yuanfudao.com](https://www.crunchbase.com/organization/yuanfudao)|EdTech|[English](https://www.pingcap.com/blog/2017-08-08-tidbforyuanfudao/); [Chinese](https://www.pingcap.com/cases-cn/user-case-yuanfudao/)|
+|[Ele.me](https://en.wikipedia.org/wiki/Ele.me)|Food Delivery|[English](https://www.pingcap.com/blog/use-case-tidb-in-eleme/); [Chinese](https://www.pingcap.com/cases-cn/user-case-eleme-1/)|
+|[LY.com](https://www.crunchbase.com/organization/ly-com)|Travel|[Chinese](https://www.pingcap.com/cases-cn/user-case-tongcheng/)||
+|[Qunar.com](https://www.crunchbase.com/organization/qunar-com)|Travel|[Chinese](https://www.pingcap.com/cases-cn/user-case-qunar/)|
+|[Hulu](https://www.hulu.com/welcome?orig_referrer=https%3A%2F%2Fwww.google.com%2F)|Entertainment|||
+|[VIPKID](https://en.wikipedia.org/wiki/VIPKID)|EdTech|||
+|[Lenovo](https://en.wikipedia.org/wiki/Lenovo)|Enterprise Technology|||
+|[Bank of Beijing ](https://en.wikipedia.org/wiki/Bank_of_China)|Banking|||
```

Comment:
The current link is for Bank of China, not Bank of Beijing. We should use this link: https://en.wikipedia.org/wiki/Bank_of_Beijing